python/ray/data/read_api.py

```diff
@@ -128,7 +128,7 @@ def from_items(items: List[Any], *, parallelism: int = -1) -> Dataset[Any]:
             run_by_consumer=False,
```

Comment:
What should be filled in for the logical_plan param?

Reply:
I was planning on leaving that off for this PR, since we can support these ops as lazy without adding support for the new planner.

Reply:
Yeah let's worry about it in 2.4.

Reply:
Makes sense to do in 2.4 since optimizer is not enabled yet.

Reply:
btw, when I say making from_XX lazy, this is exactly what I meant (it doesn't affect metadata etc.).